versions.gradle

```diff
@@ -12,8 +12,8 @@ ext {
     targetSDKVersion = 33
 
     // project versioning
-    versionCode = 4
-    versionName = '1.1.1'
+    versionCode = 6
```

Comment:
why does this go up 2?

Reply:
oops my bad. I was looking at the diff which Evan had to 5 and instead of leaving it as is incremented! Fixed this now!